shared-libs/rules-engine/src/rules-state-store.js

```diff
@@ -190,7 +195,12 @@ const self = {
   /**
    * @returns {string} User contact document
    */
-  currentUser: () => currentUser,
+  currentUserContact: () => currentUserContact,
```

Comment:
if you agree that the user attribute is no longer required on the target document, can we go one step further and eliminate this `currentUserContact` attribute?

Reply:
well, the target emits `currentUserContact`, cause the target is the one we need replicated to supervisors. (so it emits the uuid of a person contact doc that the supervisor sees). Tasks are the ones that emit the user's id (`org.couchdb.user:<username>`), so they would only be downloaded by the generating user. 
Afraid both fields are used :(